src/main/java/org/sunbird/user/registration/model/UserRegistration.java

```diff
@@ -101,6 +102,14 @@ public void setProposedDeptName(String proposedDeptName) {
 		this.proposedDeptName = proposedDeptName;
 	}
 
+	public String getUserAuthToken() {
+		return userAuthToken;
+	}
+
+	public void setUserAuthToken(String userAuthToken) {
+		this.userAuthToken = userAuthToken;
+	}
+
```

Comment:
Did we cross check the usage of userRegistration class ?
is there any side effect due to this change ?

Do test the user registration feature once this is deployed in PreProd

Reply:
@JsonIgnoreProperties(ignoreUnknown = true) is added on the entity class